.gitignore

```diff
@@ -57,6 +59,8 @@ koreader-emulator-i686-linux-gnu*
 koreader-emulator-x86_64-linux-gnu*
 koreader-emulator-x86_64-pc-linux-gnu*
 koreader-emulator-x86_64-apple-darwin*
+koreader-emulator-x86_64-unknown-linux-gnu*
```

Comment:
Is that from Clang?

Reply:
Pretty sure this is just what `_sysctl(2)` returns for my processor type:
```
$ uname --machine --processor --operating-system
x86_64 unknown GNU/Linux
```

Reply:
So do most people. ;-)

```
$ uname --machine --processor --operating-system
x86_64 unknown GNU/Linux
$ gcc -dumpmachine
x86_64-linux-gnu
$ gcc --version
gcc (Debian 8.3.0-6) 8.3.0
Copyright (C) 2018 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```

Reply:
Ok. TIL. Here's my machine:
```
$ gcc -dumpmachine
x86_64-unknown-linux-gnu
$ gcc --version
gcc (GCC) 8.3.0
$ lsb_release -a
LSB Version:    1.0
Distributor ID: VoidLinux
Description:    Void Linux
Release:        rolling
Codename:       void
```

Taking a look at my distro's gcc package, it looks to be explicitly compiling with `--target`, setting it to the above.

Reply:
On older versions of Clang (3-5, maybe also 6) it used to output something like `x86_64-unknown-linux-gnu`, but on Clang 7.0.1 it seems to be `x86_64-pc-linux-gnu`.

Reply:
But yeah, I'd just make it `koreader-emulator*` (etc. etc.) or probably even just `koreader-*`. There doesn't seem to be a good reason not to.